src/Core/Framework/Routing/Facade/RequestFacadeFactory.php

```diff
@@ -21,7 +21,8 @@ public function __construct(private readonly RequestStack $requestStack)
     public function factory(Hook $hook, Script $script): RequestFacade
     {
         $request = $this->requestStack->getMainRequest();
-        \assert($request !== null);
+        // assert disabled: null is now allowed
+        //\assert($request !== null);
```

Comment:
Not necessary to leave the former line commented out. Both comments can be removed.

Reply:
I changed and committed that.